py/server/tests/test_table_factory.py

```diff
@@ -450,5 +459,18 @@ def test_input_table_empty_data(self):
         it.delete(t)
         self.assertEqual(it.size, 0)
 
+    def test_j_input_wrapping(self):
+        cols = [
+            bool_col(name="Boolean", data=[True, False]),
+            string_col(name="String", data=["foo", "bar"]),
+        ]
+        t = new_table(cols=cols)
```

Comment:
I think we should also test that wrapping a Java Table without the attribute produces a wrapper Table and not a wrapper InputTable.